news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,206 @@
+
+## Moderation
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Kobold84.jpg)](https://osu.ppy.sh/users/3227533)
+
+Holding the line in one of the game's most unruly language-based channels, **Kobold84** is the undisputed supermoderator of the Russian-speaking community, investing thousands of hours into keeping an entire subsection of the game orderly, fair and fun for everyone.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Death.jpg)](https://osu.ppy.sh/users/3242450)
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Dntm8kmeeatu.jpg)](https://osu.ppy.sh/users/5428812)
+
+Combined, **Death and Dntm8kmeeatu** put the support in the Support Team, collectively responsible for solving literally thousands of problems and answering questions for those in need, all while balancing their duties as members of the Global Moderation Team.
+
+## Development
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Milan.jpg)](https://osu.ppy.sh/users/1052994)
+
+A chief contributor to the Mappers' Guild and Beatmap Nominator infrastructure, **Milan-**'s advice and aid have helped shape the backbone of the systems that govern osu!'s ranking cycle, and one of its primary projects for promoting the use of officially licensed music, all while maintaining his place in the Beatmap Nominators.
+
+## Nominators
+
+In an effort to acknowledge the people who form the spine of osu!'s ranking system, we will be awarding a new form of Elite badges each year to commemorate the contributions of that year's most impressive Beatmap Nominators.
+
+<div align="center">
+    <img src="https://assets.ppy.sh/profile-badges/elite-nominator.png"/>
+</div>
+
+Only the most engaged qualify for such an accolade, those who truly stand at the top of their cohort as inarguably the most active and engaged Beatmap Nominators of all.
+
+To acknowledge these efforts, the Nomination Assessment Team have come together to write the following accolades describing what each of the Elite Nominators have done.
+
+### osu!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mordred.jpg)](https://osu.ppy.sh/users/7265097)
+
+**Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his persistence goes to show how he deserves the title of Elite Nominator.
```

Comment:
```suggestion
**Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his dedication goes to show how he deserves the title of Elite Nominator.
```
"Persistence" seems like an odd choice of words, given there's no opposition of sorts mentioned in the text.